dotCMS/src/main/webapp/html/portlet/ext/useradmin/view_users_js_inc.jsp

```diff
@@ -386,7 +387,13 @@
 
 		var userEmail;
 		if(emailChanged){
-			var pattern=/^([a-zA-Z0-9_.-])+@([a-zA-Z0-9_.-])+\.([a-zA-Z])+([a-zA-Z])+/;
+            <%
+                 String emailRegex = Config.getStringProperty(com.dotmarketing.util.WebKeys.DOTCMS_USE_REGEX_TO_VALIDATE_EMAILS, null);
```

Comment:
1. Is the new pattern more conformant to the email spec? If so, it should be our default.  
2. the default value should be accessed via the getStringProperty, not a null which then gets a null check, e.g.:
```
 String emailRegex = Config.getStringProperty(com.dotmarketing.util.WebKeys.DOTCMS_USE_REGEX_TO_VALIDATE_EMAILS, "^([a-zA-Z0-9_.-])+@([a-zA-Z0-9_.-])+\.([a-zA-Z])+([a-zA-Z])+");
```